syntax = "proto3";
package example;

option cc_generic_services = true; // 需要继承RPC服务类就需要cc_generic_services设置开启才能生成

message EchoRequest {
    string message = 1;
}

message EchoResponse {
    string message = 1;
}

// 定义RPC远端方法
service EchoService {
    rpc Echo(EchoRequest) returns (EchoResponse);
}